### ISSUE TITLE
Add a time delta argument to process().

### ISSUE THREAD

**Issue by PJB3005** [d3athrow/vgstation13#10815]
Every decent game engine has such a thing on periodic update functions. Why don't we?

It would make it possible to make the various processes change speed without actually affecting game balance.

**Comment by clusterfack** [comment]
Yeah, I've thought about the fact that we should have this. We need it for some things I want to do in the future.

**Comment by gbasood** [comment]
How would this be done?
world.time comparisons for every process()?

**Comment by clusterfack** [comment]
No, pass it as an argument to all process()es

**Comment by gbasood** [comment]
No, I'm talking about how we actually calculate delta time

**Comment by PJB3005** [comment]
Scheduler might already have a var for that actually.

**Comment by clusterfack** [comment]
Make a var for the last scheduler process world time and store it. Then at the start of dowork and each time scheck finishes recalculate the delta time.

**Comment by ghost** [comment]
> "Every decent game engine has such a thing on periodic update functions. Why don't we?"

Well
